release/artifacts.yaml

```diff
@@ -172,6 +172,8 @@
   mavenUpload: True
 - target: //ledger/sandbox:ledger-api-server
   type: jar-scala
```

Comment:
This removed the `mavenUpload` property from ledger-api-server.

Reply:
oops!